modules/030-cloud-provider-gcp/openapi/openapi-case-tests.yaml

```diff
@@ -1,20 +1,4 @@
-defaults:
-
-positive:
-  values:
-    - internal:
-        providerClusterConfiguration:
-          sshKey: test
-          provider:
-            region: test
-            serviceAccountJSON: test
-        providerDiscoveryData:
-          disableExternalIP: true
-          instances:
-            diskSizeGb: 50
-            diskType: test
-            image: image
-            networkTags: ["test1"]
-          networkName: test
-          subnetworkName: test
-          zones: ["test"]
```

Comment:
why we remove values ?

Reply:
I took an example from here: https://github.com/deckhouse/deckhouse/blob/main/modules/030-cloud-provider-azure/openapi/openapi-case-tests.yaml